res/res_agi.c

```diff
@@ -945,6 +1059,9 @@
 		</see-also>
 	</agi>
 	<agi name="speech create" language="en_US">
+		<since>
```

Comment:
The speech stuff came in 1.6.0 I believe

Reply:
I might have to re-think the approach.  This file has had several major refactors so it's very hard to tell when something appeared.